### ISSUE TITLE
Admin UI refreshes on file change in public folder

### ISSUE THREAD

**Issue by boazpoolman** [strapi/strapi#11645]
### Describe the bug

Whenever I change a file that's located in the public folder of my Strapi instance the admin UI in my browser refreshes.
Like a hard refresh, not like a fancy SPA refresh 😛

It's only happening when I change an existing file. When I create a new file the Admin does not refresh.

Issue seems to have started happening since version `4.0.0-beta.12`.

### Steps to reproduce the behavior

To reproduce the bug you can create a file in the public folder and afterwards make a change to its content.
Another way, that's probably more likely to really happen when using Strapi, is by creating a media item in the media library and afterwards changing the file of the media item by clicking `Replace media`.

### Expected behavior

For the Admin UI not to refresh

### System

- Node.js version: v14.7.0
- NPM version: 6.14.7
- Strapi version: 4.0.0-beta.14
- Database: Postgres
- Operating system: Mac OS

**Comment by derrickmehaffy** [comment]
Can't reproduce on Linux with beta.14, @boazpoolman can you record a loom?

**Comment by boazpoolman** [comment]
Note:
This only occurs when running the admin panel with the `--watch-admin` flag.
Eventhough it is added to the `watchIgnoreFiles` list by default: https://github.com/strapi/strapi/blob/releases/v4/packages/core/strapi/lib/commands/develop.js#L135

**Comment by derrickmehaffy** [comment]
Yup there I can, only the frontend dev server is restarting not the backend.
